velox/connectors/hive/storage_adapters/abfs/AbfsConfig.h

```diff
@@ -49,11 +66,42 @@ class AbfsConfig {
     return filePath_;
   }
 
+  std::string connectionString() const {
+    return connectionString_;
+  }
+
+  std::string url() const {
+    return url_;
+  }
+
+  std::string urlWithSasToken() const {
```

Comment:
These APIs are only used for testing. Can we remove these, create the read/write clients here, and then use something like GetAccountInfo from the client object in the tests?
https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_client.hpp

Reply:
Only available function is `getUrl`. Others like `GetAccountInfo` need to retrieve info from remote server.